src/Loggers/LogEmbeddings.jl

```diff
@@ -0,0 +1,58 @@
+"""
+
+"""
+function log_embeddings(logger::TBLogger, name::AbstractString, mat::AbstractArray; metadata=nothing, labels=nothing, metadata_header=nothing, step=nothing)
+    @assert ndims(mat) == 2 "Embedding matrix must be 2-D"
+    matrix_path = joinpath(logger.logdir, string(step), name)
+    mkpath(matrix_path)
+    if metadata != nothing
+        @assert length(metadata) == size(mat, 1) "#labels must be equal to #samples"
+        if metadata_header == nothing
+            metadata_header = [string(x) for x in metadata]
+        else
+            @assert length(metadata_header) == size(metadata, 2) "length of header must be equal to the number of columns in metadata"
+            metadata = vcat([join(metadata_header, '\t')], [join(x, '\t') for x in metadata])
```

Comment:
Do you think the following might be clearer:
```suggestion
            metadata = [join(metadata_header, '\t'); join(x, '\t') for x in metadata])
```